src/docfx/config/Config.cs

```diff
@@ -77,10 +77,15 @@ internal class Config
         public readonly RouteConfig[] Routes = Array.Empty<RouteConfig>();
 
         /// <summary>
-        /// Gets the configuration about contributors.
+        /// Gets the configuration about contributors display in page.
         /// </summary>
         public readonly ContributorConfig Contributor = new ContributorConfig();
```

Comment:
can we merge these 2 into 1 section `Contribution`?

Reply:
Sure. Contribution.Enabled to control whether to show "edit button", and use other config to control whether to display author/contributors.